### ISSUE TITLE
[Question] About the derivitive mode. How I pass the derivative to the optimizer if I write the objective function with a autodifferentiation code?

### ISSUE THREAD

**Issue by floatingCatty** [wujian16/Cornell-MOE#93]
Hello!
Thanks for providing such a powerful library.

I am currently implementing some optimization algorithm that combine the gradient descent and BO.
And the code is written with pytorch, so, in each evaluation, I can get the gradient of the loss w.r.t. the parameters.
How can I pass the gradient to the BO optimizer with derivatives to accelerate its convergence ?

Thanks a lot for your time.

**Comment by MaAl13** [comment]
I have the same problem
